api/app/settings/common.py

```diff
@@ -645,6 +646,26 @@
     },
 }
 
+REDIS_CACHE_NAME = "redis-cache"
+
+# Log exceptions generated by django-redis
+DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS = env.bool(
+    "DJANGO_REDIS_LOG_IGNORED_EXCEPTIONS", True
+)
+
+# Uses redis-py native URL notation
```

Comment:
Can we add an e.g. here?

Reply:
I think maybe this got lost when you refactored the settings, although I guess it technically doesn't have a place here anymore. We should add some documentation somewhere though perhaps? I know we need to add SaaS based user facing documentation but we also need to document it from a self hosted perspective - we can add an example redis URL there.